sync.js

```diff
@@ -1,13 +1,13 @@
 /**
```

Comment:
I noticed that this file should be transcode to typescript, too. Is it possible to do that within this PR?

Reply:
I'll do this in another PR because we have another todo which is required for the next release. That is a fix for container image.